operator/pkg/utils.go

```diff
@@ -0,0 +1,13 @@
+package operator
+
+import (
+	"math/big"
+
+	"github.com/yetanotherco/aligned_layer/common"
+)
+
+func IsVerifierDisabled(disabledVerifiersBitmap *big.Int, verifierId common.ProvingSystemId) bool {
+	verifierIdInt := uint8(verifierId)
+	bit := disabledVerifiersBitmap.Uint64() & (1 << verifierIdInt)
```

Comment:
If the bitmap is uint256, I think it will not fit in a uint64

Reply:
Yes, that's correct. The thing is that to perform the bitwise `and` operation, I need to cast the big integer into a `uint64`. We should never have a bitmap larger than 64 bits, since we’ll likely never have more than 63 proving systems. If we accidentally set a bitmap larger than expected, the function will wrap the number and return 0, meaning no verifier will be disabled. I’m not sure if this is the intended behavior, but it seems acceptable.